File: ControlRemote/Web/ClientApp/src/app/Dto/EmployerModel.ts
export class EmployerModel {
    id: number;
    managerId: number;
    name: string;
    login: string;

    public constructor(_id: number, _managerId: number, _name: string, _login: string) {
        this.id = _id;
        this.managerId = _managerId;
        this.name = _name;
        this.login = _login;
    }
}

File: ControlRemote/Web/Dto/EmployerModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Web.Dto
{
    public class EmployerModel
    {
        public int Id { get; set; }
        public int ManagerId { get; set; }
        public string Name { get; set; }
        public string Login { get; set; }
    }
}
